libs/generators/src/lib/generators/generators-menu/generators-menu.component.spec.ts

```diff
@@ -19,4 +27,24 @@ describe('GeneratorsMenuComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('navigate()', () => {
+    it('Should navigate to generator with provided generator name when function was called', () => {
+      jest.spyOn(routerMock, 'navigate');
+      component.navigate('generator_name_mock');
+      expect(routerMock.navigate).toHaveBeenCalledWith([
+        'generators',
+        'generator_name_mock',
+      ]);
+    });
+  });
+
+  describe('Menu Item', () => {
+    it('Should navigate to generator when menu item was clicked', () => {
+      jest.spyOn(component, 'navigate');
+      const menuItem = fixture.debugElement.query(By.css('.menu-item'));
+      menuItem.triggerEventHandler('click', null);
+      expect(component.navigate).toHaveBeenCalled();
+    });
+  });
```

Comment:
These tests are redundant after you'll use routerLink like @OmerGronich suggested